ccd-definition/CaseEventToFields/CareSupervision/uploadAdditionalDocuments-nonprod.json

```diff
@@ -0,0 +1,144 @@
+[
```

Comment:
is the name of file correct? event is uploadAdditionalApplications, file name says about documents

Reply:
sorry changed this